hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/AuthAwareTokenConverter.kt

```diff
@@ -55,10 +55,10 @@ private fun extractAuthorities(jwt: Jwt, converter: Converter<Jwt, Collection<Gr
 
 class AuthAwareAuthenticationToken(
   jwt: Jwt,
-  val userName: String?,
   val clientId: String,
+  val userName: String? = null,
   val authSource: AuthSource = AuthSource.NONE,
-  authorities: Collection<GrantedAuthority>,
+  authorities: Collection<GrantedAuthority> = emptyList(),
```

Comment:
Add parameter defaults for nullables and put them at the end of the parameter list.